### ISSUE TITLE
Auto-select first menu item

### ISSUE THREAD

**Issue by marisademeglio** [daisy/pipeline-ui#186]
From recent tester feedback:

"when open any menu, first menu item inside the menu should be selected"

**Comment by marisademeglio** [comment]
Is this standard behavior on Windows? @NPavie can you see what happens by default if you alt-F to the file menu in the Pipeline? On mac, the Pipeline behavior matches other native apps, e.g. mac Mail.

**Comment by NPavie** [comment]
@marisademeglio i checked on several app, and it depends : 
- on Chrome (and chrome-based app that have menus), indeed no menu item are selected by default when opening menus
- but on Microsoft applications (including edge, VSCode, but also word, visual studio community), the first item is automatically selected on opening the menu

I assume that on Windows, on activating the menu, we will need to manually set focus on the first element on opening.

i'll look into that in the app.

**Comment by NPavie** [comment]
hum this issue might be a bit difficult to solve

I did not find a way so far to force the focus on the first element of the opening menu on opening the menu.

I might look at how vscode does it but i think they overloaded the menu bar creation.

**Comment by NPavie** [comment]
Opened an issue regarding this in https://github.com/electron/electron/issues/41324

**Comment by NPavie** [comment]
I did some experiment on electron dev requests as they were not able to reproduce the issue right away and reported to them the resultsts : 
It seems this selection issue happens when the focus or the mouse is on the web view.

When selecting the application outside the web view (like by switching to the app with alt-tab), the first item of the menu is correctly selected on opening the menu.

**Comment by avneeshsingh** [comment]
The deadline for final deliverable is close. So, it is OK to defer such minor issues that are taking considerable time. These can be fixed in future releases.

**Comment by marisademeglio** [comment]
It seems like this has been fixed in newer versions of Electron. I cannot reproduce it anymore. Opening the menus via keyboard selects the first available menu item.
